src/view/hooks/useYfmHtmlBlockStyles.ts

```diff
@@ -0,0 +1,41 @@
+import {useEffect, useState} from 'react';
+
+import {useThemeValue} from '@gravity-ui/uikit';
+
+import {IHTMLIFrameElementConfig} from '../../extensions';
+import {getYfmHtmlBlockCssVariables} from '../hocs/withYfmHtml/utils';
+
+export const useYfmHtmlBlockStyles = () => {
+    const theme = useThemeValue();
+    const [config, setConfig] = useState<IHTMLIFrameElementConfig | undefined>();
+
+    useEffect(() => {
+        const bodyStyles = window.getComputedStyle(document.body);
+
+        const colorTextPrimary = bodyStyles.getPropertyValue('--g-color-text-primary');
+        const colorTextSecondary =
+            theme === 'light' || theme === 'light-hc'
+                ? bodyStyles.getPropertyValue('--yfm-color-hljs-subst')
```

Comment:
??

Reply:
I used the style for [textarea](https://github.com/gravity-ui/markdown-editor/blob/4918caf4c2d86f2b5b1db71be5f60ca3010ca005/src/extensions/yfm/YfmHtmlBlock/YfmHtmlBlockNodeView/YfmHtmlBlock.scss#L56), I'll fix it